packages/terra-framework-docs/CHANGELOG.md

```diff
@@ -7,14 +7,14 @@
 
 * Fixed
   * Fixed the `terra-tabs` icon-only examples keyboard navigation issue.
+  * Fixed broken links in the Worklist Data Grid documentation in `terra-data-grid` component.
 
 * Added
   * Added test for data grid column resizing.
   * Added example for Flowsheet Data Grid containing "No Result" cells.
 
-* Fixed
-  * Fixed the `terra-tabs` icon-only examples keyboard navigation issue.
-  * Fixed broken links in the Worklist Data Grid documentation in `terra-data-grid` component.
+* Updated
+  * Updated `terra-table` example to show providing custom masked cell message label.
```

Comment:
Can we replace "show" with "announce" probably? As it's something only being announced with the screen readers and not being visible.

Reply:
I was stating that the example shows how to use this new feature.  In the implementation there is an announcement, yes.